stdlib/2and3/array.pyi

```diff
@@ -7,27 +7,30 @@ from typing import (Any, BinaryIO, Generic, Iterable, Iterator, List, MutableSeq
                     overload, Text, Tuple, TypeVar, Union)
 from typing_extensions import Literal
 
-IntTypeCodes = Literal['b', 'B', 'h', 'H', 'i', 'I', 'l', 'L', 'q', 'Q']
-FloatTypeCodes = Literal['f', 'd']
-UnicodeTypeCodes = Literal['u']
-TypeCodes = Literal[IntTypeCodes, FloatTypeCodes, UnicodeTypeCodes]
+IntTypeCode = Literal['b', 'B', 'h', 'H', 'i', 'I', 'l', 'L', 'q', 'Q']
```

Comment:
Actually these should be prefixed with underscores to indicate that they don't exist at runtime. Sorry for missing this the first time!